package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "tsup",
     "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
```

Comment:
last I remember, file extensions had to be provided as cli flags to extend the defaults, using the --ext (IIRC) flag

Reply:
Hi Simone, you are right, but based on my tests, ESLint works correctly on both JS and TS files without setting `--ext`. This is explained here: https://github.com/typescript-eslint/typescript-eslint/issues/3824

In summary:
The `--ext` default value is "_`.js` and the files that match the `overrides` entries of your configuration_" ([eslint doc](https://eslint.org/docs/latest/use/command-line-interface#--ext)) and the `@typescript-eslint/recommended` adds the `*.ts` and `*.tsx` extensions to the `overrides` entries.

Reply:
ha, makes sense, thanks 👌